source/integrate/jira-interoperability.rst

```diff
@@ -0,0 +1,298 @@
+- ``updated_prioity=1``: updated issue priority
+- ``updated_rank=1``: ranked issue higher or lower
+- ``updated_sprint=1``: assigned issue to a different sprint
+- ``updated_status=1``: transitioned issed to a different status, such as Done or In Progress
+- ``updated_summary=1``: renamed issue
+
+Here's an example of a webhook configured to create a post for comment events: ``https://community.mattermost.com/plugins/jira/webhook?secret=MYSECRET&team=contributors&channel=town-square&updated_comments=1``
+
+.. tip::
+  
+  Any previously configured webhooks set up in Jira that point to specific channels are supported and will continue to work.
+
+Manage channel subscriptions in Mattermost
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The Channel Subscription modal provides easy access for Mattermost channel admins to set up which notifications they want to receive per channel. In any channel, run the ``jira /subscribe`` slash command, and configure the following options:
+
+- Configure what Jira notifications are sent to the current channel.
+- Specify filters including: affects versions, epic link, fix versions, labels, and priority.
+- Specify custom fields including: checkboxes, labels, radio buttons, and select list (single or multiple choice). 
+- Review the approximate JQL output generated. This is not guaranteed to be valid JQL and is only shown as a reference to what the query may look like if converted to JQL.
+
+Run the ``/jira subscribe list`` slash command to display all subscription rules set up across all channels and teams on your Mattermost instance.
+
+.. note::
+
+  - A user must meet the criteria of both the Mattermost user settings and Jira group settings in order to edit subscriptions. If you can subscribe channels to Jira events, you can also set up rules that define when a particular event with certain criteria are met in Jira that trigger a notification is sent to a particular channel.
+  - These subscription rules can specify the Jira Project, Event Type, Issue Type, and can filter out issues with certain values. 
+  - When a user is setting up a notification subscription, they'll only see the projects and issue types they have access to within Jira. If they can't see a project in Jira, it won't be displayed as an option for that particular user when they are trying to set up a subscription in Mattermost.
+  - If your organization's infrastructure is set up in such a way that your Mattermost instance can't connect to your Jira instance, channel subscriptions won't be avaialable. Instead, use `legacy Webhooks <#legacy-jira-webhooks>`__ instead to allow a Jira webhook to post to a specific channel.
+
+Manage notifications
+~~~~~~~~~~~~~~~~~~~~
+
+Jira notifications are messages sent to a Mattermost channel when a particular event occurs in Jira. They can managed as `channel subscriptions <#manage-channel-subscriptions>`__ in Mattermost, or managed as `webhooks <#configure-webhooks-in-Jira>`__ in Jira. 
```

Comment:
These two links don't seem to scroll on my browser

Reply:
I'll investigate and correct.

Reply:
Fixed